packages/@ama-sdk/core/src/fwk/Reviver.ts

```diff
@@ -7,6 +7,12 @@ export interface ReviverOptions {
    * @default console
    */
   logger?: Logger;
+
+  /**
+   * Determine if the fields from dictionary should be removed from the given object
```

Comment:
it's not clear in the description that this option only applies when `dictionaries` is undefined